homeassistant/components/sensor/hydroquebec.py

```diff
@@ -170,23 +174,26 @@ def __init__(self, username, password, contract=None):
         self._contract = contract
         self.data = {}
 
+    @asyncio.coroutine
     def get_contract_list(self):
         """Return the contract list."""
         # Fetch data
-        self._fetch_data()
+        yield from self._fetch_data()
         return self.client.get_contracts()
 
+    @asyncio.coroutine
+    @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def _fetch_data(self):
         """Fetch latest data from HydroQuebec."""
         from pyhydroquebec.client import PyHydroQuebecError
         try:
-            self.client.fetch_data()
+            yield from self.client.fetch_data()
         except PyHydroQuebecError as exp:
             _LOGGER.error("Error on receive last Hydroquebec data: %s", exp)
             return
 
-    @Throttle(MIN_TIME_BETWEEN_UPDATES)
+    @asyncio.coroutine
     def update(self):
```

Comment:
This needs to be an `async_update`

Reply:
@balloob This is not the update method directly call by hass, this method is called by an other `async_update` method.
So my question is: do coroutine names have to start by `async_` ?

Reply:
Good point! No they don't. Only the entity method needs to be named `async_update` for the code to work, if it's a coroutine. But for clarity it could be wise to have this method be called `async_update` too.